crates/server/src/admin/handlers/root_keys.rs

```diff
@@ -67,3 +67,21 @@ pub fn store_root_key(
     info!("Root key stored successfully.");
     Ok(true)
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct DeleteKeysResponse {
+    data: Empty,
+}
+pub async fn delete_auth_keys_handler(
+    Extension(state): Extension<Arc<AdminState>>,
+) -> impl IntoResponse {
+    clean_auth_keys(&state.store).map_or_else(
+        |err| parse_api_error(err).into_response(),
+        |_| {
+            ApiResponse {
+                payload: DeleteKeysResponse { data: Empty {} },
+            }
+            .into_response()
```

Comment:
..or better yet, just `().into_response()` for an empty body, status code 200, no content-type

Reply:
I don't want empty body. Return structure is either data or error

Reply:
Why not boolean true?

Reply:
There is no current case that will return false but not error so it does not bring any value